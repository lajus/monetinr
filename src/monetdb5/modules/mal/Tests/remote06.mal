# let connect figure out itself how to connect to the running db
uri := sabaoth.getLocalConnectionURI();

# create a persistent connecton to ourself
conn1:str := remote.connect(uri, "monetdb", "monetdb");
# do it again ...
conn2:str := remote.connect(uri, "monetdb", "monetdb");
# ... and check they are not the same
barrier if := conn1 == conn2;
	io.printf("!oops: %s == %s\n", conn1, conn2);
exit if;

b := bat.new(:oid,:int);

bat.append(b, 1);
rb1 := remote.put(conn1, b);

bat.append(b, 3);
rb2 := remote.put(conn2, b);

lb1:bat[:oid,:int] := remote.get(conn1, rb1);
lb2:bat[:oid,:int] := remote.get(conn2, rb2);

io.print(lb1);
io.print(lb2);

# help testweb a bit, since currently no cleanup is done on server
# shutdown
remote.disconnect(conn1);
remote.disconnect(conn2);
