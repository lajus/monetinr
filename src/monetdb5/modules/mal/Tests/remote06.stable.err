stderr of test 'remote06` in directory 'monetdb5/modules/mal` itself:


# 23:22:52 >  
# 23:22:52 >   mserver5 "--config=/export/scratch/stripe/fabian/monetdb/current/program-i86pc/etc/monetdb5.conf" --debug=10 --set gdk_nr_threads=0 --set "monet_mod_path=/export/scratch/stripe/fabian/monetdb/current/program-i86pc/lib/MonetDB5:/export/scratch/stripe/fabian/monetdb/current/program-i86pc/lib/MonetDB5/lib:/export/scratch/stripe/fabian/monetdb/current/program-i86pc/lib/MonetDB5/bin" --set "gdk_dbfarm=/export/scratch/stripe/fabian/monetdb/current/mtest-pegasus/five/dbfarm" --set "sql_logdir=/export/scratch/stripe/fabian/monetdb/current/mtest-pegasus/five/sqllog" --set "xquery_logdir=/export/scratch/stripe/fabian/monetdb/current/mtest-pegasus/five/xquerylog" --set mapi_open=true --set xrpc_open=true --set mapi_port=39169 --set xrpc_port=43287 --set monet_prompt= --trace  --dbname=mTests_src_modules_mal  remote06.mal
# 23:22:52 >  

# builtin opt 	gdk_arch = 64bitx86_64-pc-solaris2.11
# builtin opt 	gdk_version = 1.27.0
# builtin opt 	monet_pid = 25433
# builtin opt 	prefix = /export/scratch/stripe/fabian/monetdb/current/program-i86pc
# builtin opt 	exec_prefix = ${prefix}
# builtin opt 	gdk_dbname = tst
# builtin opt 	gdk_dbfarm = ${prefix}/var/MonetDB
# builtin opt 	gdk_debug = 8
# builtin opt 	gdk_mem_bigsize = 262144
# builtin opt 	gdk_alloc_map = yes
# builtin opt 	gdk_mem_pagebits = 14
# builtin opt 	gdk_vmtrim = yes
# builtin opt 	monet_admin = adm
# builtin opt 	monet_prompt = >
# builtin opt 	monet_welcome = yes
# builtin opt 	monet_mod_path = /export/scratch/stripe/fabian/monetdb/current/program-i86pc/lib/MonetDB:/export/scratch/stripe/fabian/monetdb/current/program-i86pc/lib/bin
# builtin opt 	monet_daemon = yes
# builtin opt 	host = localhost
# builtin opt 	mapi_port = 50000
# builtin opt 	mapi_noheaders = no
# builtin opt 	mapi_debug = 0
# builtin opt 	mapi_clients = 2
# builtin opt 	sql_debug = 0
# builtin opt 	sql_logdir = ${prefix}/var/MonetDB
# builtin opt 	xquery_logdir = ${prefix}/var/MonetDB
# builtin opt 	standoff_ns = http://monetdb.cwi.nl/standoff
# builtin opt 	standoff_start = start
# builtin opt 	standoff_end = end
# config opt 	prefix = /export/scratch/stripe/fabian/monetdb/current/program-i86pc
# config opt 	config = ${prefix}/etc/monetdb5.conf
# config opt 	prefix = /export/scratch/stripe/fabian/monetdb/current/program-i86pc
# config opt 	exec_prefix = ${prefix}
# config opt 	gdk_dbfarm = ${prefix}/var/MonetDB5/dbfarm
# config opt 	gdk_dbname = demo
# config opt 	gdk_alloc_map = no
# config opt 	gdk_embedded = no
# config opt 	gdk_debug = 0
# config opt 	monet_mod_path = /export/scratch/stripe/fabian/monetdb/current/program-i86pc/lib/MonetDB5:/export/scratch/stripe/fabian/monetdb/current/program-i86pc/lib/MonetDB5/lib:/export/scratch/stripe/fabian/monetdb/current/program-i86pc/lib/MonetDB5/bin
# config opt 	monet_daemon = no
# config opt 	monet_welcome = yes
# config opt 	mero_msglog = ${prefix}/var/log/merovingian.log
# config opt 	mero_errlog = ${prefix}/var/log/merovingian.log
# config opt 	mero_timeinterval = 600
# config opt 	mero_pidfile = ${prefix}/var/run/merovingian.pid
# config opt 	mero_exittimeout = 7
# config opt 	mero_doproxy = yes
# config opt 	mero_discoveryttl = 600
# config opt 	mal_init = /export/scratch/stripe/fabian/monetdb/current/program-i86pc/lib/MonetDB5/mal_init.mal
# config opt 	mal_listing = 2
# config opt 	checkpoint_dir = ${prefix}/var/MonetDB5/chkpnt
# config opt 	mapi_port = 50000
# config opt 	mapi_open = false
# config opt 	sql_logdir = ${prefix}/var/MonetDB5/sql_logs
# config opt 	sql_init = /export/scratch/stripe/fabian/monetdb/current/program-i86pc/lib/MonetDB5/sql_init.sql
# cmdline opt 	config = /export/scratch/stripe/fabian/monetdb/current/program-i86pc/etc/monetdb5.conf
# cmdline opt 	gdk_nr_threads = 0
# cmdline opt 	monet_mod_path = /export/scratch/stripe/fabian/monetdb/current/program-i86pc/lib/MonetDB5:/export/scratch/stripe/fabian/monetdb/current/program-i86pc/lib/MonetDB5/lib:/export/scratch/stripe/fabian/monetdb/current/program-i86pc/lib/MonetDB5/bin
# cmdline opt 	gdk_dbfarm = /export/scratch/stripe/fabian/monetdb/current/mtest-pegasus/five/dbfarm
# cmdline opt 	sql_logdir = /export/scratch/stripe/fabian/monetdb/current/mtest-pegasus/five/sqllog
# cmdline opt 	xquery_logdir = /export/scratch/stripe/fabian/monetdb/current/mtest-pegasus/five/xquerylog
# cmdline opt 	mapi_open = true
# cmdline opt 	xrpc_open = true
# cmdline opt 	mapi_port = 39169
# cmdline opt 	xrpc_port = 43287
# cmdline opt 	monet_prompt = 
# cmdline opt 	gdk_dbname = mTests_src_modules_mal
# cmdline opt 	gdk_debug = 10
#warning: please don't forget to set your vault key!
#(see /export/scratch/stripe/fabian/monetdb/current/program-i86pc/etc/monetdb5.conf)

# 23:22:53 >  
# 23:22:53 >  Done.
# 23:22:53 >  

