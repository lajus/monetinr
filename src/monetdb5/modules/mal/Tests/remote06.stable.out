stdout of test 'remote06` in directory 'monetdb5/modules/mal` itself:


# 20:53:23 >  
# 20:53:23 >   mserver5 "--config=/export/scratch/stripe/fabian/monetdb/current/program-i86pc/etc/monetdb5.conf" --debug=10 --set gdk_nr_threads=0 --set "monet_mod_path=/export/scratch/stripe/fabian/monetdb/current/program-i86pc/lib/MonetDB5:/export/scratch/stripe/fabian/monetdb/current/program-i86pc/lib/MonetDB5/lib:/export/scratch/stripe/fabian/monetdb/current/program-i86pc/lib/MonetDB5/bin" --set "gdk_dbfarm=/export/scratch/stripe/fabian/monetdb/current/mtest-pegasus/five/dbfarm" --set "sql_logdir=/export/scratch/stripe/fabian/monetdb/current/mtest-pegasus/five/sqllog" --set "xquery_logdir=/export/scratch/stripe/fabian/monetdb/current/mtest-pegasus/five/xquerylog" --set mapi_open=true --set xrpc_open=true --set mapi_port=36372 --set xrpc_port=44624 --set monet_prompt= --trace  --dbname=mTests_src_modules_mal  remote06.mal
# 20:53:23 >  

# MonetDB server v5.9.0, based on kernel v1.27.0
# Serving database 'mTests_src_modules_mal', using 1 thread
# Compiled for x86_64-pc-solaris2.11/64bit with 64bit OIDs dynamically linked
# Copyright (c) 1993-July 2008 CWI.
# Copyright (c) August 2008- MonetDB B.V., all rights reserved
# Visit http://monetdb.cwi.nl/ for further information
# Listening for connection requests on mapi:monetdb://pegasus:36372/
function user.main():void;
# let connect figure out itself how to connect to the running db 
    uri := sabaoth.getLocalConnectionURI();
# create a persistent connecton to ourself 
    conn1:str  := remote.connect(uri,"monetdb","monetdb");
# do it again ... 
    conn2:str  := remote.connect(uri,"monetdb","monetdb");
# ... and check they are not the same 
barrier if := calc.==(conn1,conn2);
    io.printf("!oops: %s == %s\n",conn1,conn2);
exit if;
    b := bat.new(:oid,:int);
    bat.append(b,1);
    rb1 := remote.put(conn1,b);
    bat.append(b,3);
    rb2 := remote.put(conn2,b);
    lb1:bat[:oid,:int]  := remote.get(conn1,rb1);
    lb2:bat[:oid,:int]  := remote.get(conn2,rb2);
    io.print(lb1);
    io.print(lb2);
# help testweb a bit, since currently no cleanup is done on server 
# shutdown 
    remote.disconnect(conn1);
    remote.disconnect(conn2);
end main;
#-----------------#
# h	t	  # name
# int	int	  # type
#-----------------#
[ 0@0,	  1	  ]
#-----------------#
# h	t	  # name
# int	int	  # type
#-----------------#
[ 0@0,	  1	  ]
[ 1@0,	  3	  ]

# 20:53:23 >  
# 20:53:23 >  Done.
# 20:53:23 >  

